### PR TITLE
Adds hidden clearable manual facet to all content finder

### PR DESCRIPTION
**Pull request by oscarwyatt**
Adds hidden clearable manual facet to all content finder

Trello: https://trello.com/b/yCngBbCU/bravigation-doing-q4-q3

### CODE REVIEW

File: config/finders/all_content.yml [hunk 1]
```diff
@@ -31,9 +25,7 @@ details:
     name: Manual
     preposition: in manual
     short_name: in
-    type: hidden
-    show_option_select_filter: false
-    allowed_values: []
+    type: hidden_clearable
```
Comment (sihugh) [inline config/finders/all_content.yml:34]: Isn't this the only change we need?  I think the other changes look more like a frontend schema - for example we don't need the supertypes, and we definitely do need the routes.

In other news, I noticed yesterday that the finderpresenter tests aren't running properly, so they're not picking up schema violations where they should here: https://github.com/alphagov/rummager/blob/master/spec/unit/content_item_publisher/finder_presenter_spec.rb#L30.  I was going to have a poke to fix them.

**Review by sihugh — CHANGES_REQUESTED** [review]
I'm not sure this is quite right - we can't directly translate a frontend schema to a backend schema, sadly.

File: config/finders/all_content.yml (hunk 1 above)
Comment (oscarwyatt) [reply config/finders/all_content.yml:34]: Apologies, act in haste, repent at leisure, fixed now

**Review by sihugh — APPROVED** [review]

File: config/finders/all_content.yml (hunk 1 above)
Comment (sihugh) [reply config/finders/all_content.yml:34]: We've all done it 🤦‍♂️

**Pull request merged**